addon/background.js

```diff
@@ -17,7 +17,7 @@ const ga = new TestPilotGA({
   av: manifest.version,
   cd19: "dev", // could be: local, dev, stage, or production
   ds: "addon",
-  tid: "", // production value is "UA-77033033-7"
+  tid: buildSettings.NODE_ENV === "prod" ? "UA-77033033-7" : "",
```

Comment:
I'd have to dig around what `cd19` dimension is referring to, but do we care if it is "production" but our `NODE_ENV` is "prod"? Not sure if they have to be in sync, but I guess the only thing that matters is whomever is building the XPI has the proper env value set.

Reply:
ah, true that